tools/postinstall-android.js

```diff
@@ -108,20 +110,27 @@ shelljs.cp('-R', path.join(appProjectRoot, 'app', 'src', 'main', 'AndroidManifes
 
 // Replacing values in top level build.gradle to avoid conflicts in Gradle builds.
 console.log('Fixing project workspace build.gradle');
-var oldGradleToolsVersion = "com.android.tools.build:gradle:3.0.0";
-var newGradleToolsVersion = "com.android.tools.build:gradle:3.2.0";
+var oldGradleToolsVersion = "com.android.tools.build:gradle:3.0.1";
+var newGradleToolsVersion = "com.android.tools.build:gradle:3.2.1";
 replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), oldGradleToolsVersion, newGradleToolsVersion);
+replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), '27.0.1', '28.0.3');
 replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), 'defaultMinSdkVersion=19', 'defaultMinSdkVersion=21');
-replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), 'defaultTargetSdkVersion=26', 'defaultTargetSdkVersion=28');
-replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), 'defaultCompileSdkVersion=26', 'defaultCompileSdkVersion=28');
+replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), 'defaultTargetSdkVersion=27', 'defaultTargetSdkVersion=28');
+replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), 'defaultCompileSdkVersion=27', 'defaultCompileSdkVersion=28');
+
+// Fixing AndroidManifest.xml to work with new versions of Android Studio.
+var usesSdkTagRegex = /<uses-sdk.*>/g;
+replaceTextInFile(path.join(appProjectRoot, 'app', 'AndroidManifest.xml'), usesSdkTagRegex, '');
```

Comment:
Removes the `<uses-sdk...` tag from `AndroidManifest.xml` until `Cordova` fixes this down the road.

Reply:
It's set via `build.gradle` anyway.